### PR TITLE
Update Q7 for GHAS

### PR DESCRIPTION
**Pull request by garysassano**
## PR Type

- [x] Content changes (add/update questions, answers, explanations)
- [ ] Bugfix
- [ ] Feature
- [ ] Refactor (no functional changes)
- [ ] Documentation changes
- [ ] Other

## What's new?
Current question has two possible correct answers and not too much clear.

### CODE REVIEW

**Review by FidelusAleksander — COMMENTED** [review]
Hi @garysassano 

1.  I'm not sure which other answer You think is correct aswell in the current question? I personally think the current state is correct, but curious to hear Your view.
2. This is a whole question rewrite, instead add a new question with what You just wrote please.

**Comment by garysassano** [comment]
This option is also correct: `Secret scanning scans your code for potential vulnerabilities that could expose secrets such as private keys or tokens.`

It's a bit generic since it says `code` instead of `Git history on all branches` but still correct overall.

**Comment by garysassano** [comment]
This option is also correct: `Secret scanning scans your code for potential vulnerabilities that could expose secrets such as private keys or tokens.`

It's a bit generic since it says `code` instead of `Git history on all branches`, yet still correct overall.

**Comment by garysassano** [comment]
@FidelusAleksander Let's make the answer options less ambiguous.

File: content/questions/advanced_security/question-007.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ draft: false
...
 
 > https://docs.github.com/en/code-security/secret-scanning/about-secret-scanning
-1. [x] Secret scanning will scan your GitHub repository for secrets such as private keys or tokens.
-1. [ ] Secret scanning is a tool for secure secret storage and management.
-1. [ ] Secret scanning is a git hook that will scan your commits for secrets such as private keys or tokens before they are pushed to GitHub.
-1. [ ] Secret scanning scans Your code for potential vulnerabilities that could expose secrets such as private keys or tokens.
+1. [x] Secret scanning scans your repository for secrets such as private keys or tokens.
+2. [ ] Secret scanning scans your repository for potential code vulnerabilities that could expose secrets such as private keys or tokens.
+3. [ ] Secret scanning is a tool for secure secret storage and management.
+4. [ ] Secret scanning is a git hook that will scan your commits for secrets such as private keys or tokens before they are pushed to GitHub.
```
Comment (FidelusAleksander) [inline content/questions/advanced_security/question-007.md:12]:
```suggestion
1. [ ] Secret scanning scans your repository for potential code vulnerabilities that could expose secrets such as private keys or tokens.
1. [ ] Secret scanning is a tool for secure secret storage and management.
1. [ ] Secret scanning is a git hook that will scan your commits for secrets such as private keys or tokens before they are pushed to GitHub.
```

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
Small but important change, then it's ready to merge

**Comment by FidelusAleksander** [comment]
@garysassano 

https://github.com/FidelusAleksander/githubcertified/pull/128/commits/de9770f257133e4c6f795ee15f53cad3c6f0899f

This commit was good, but You changed it again into a multiple choice question syntax :)

**Comment by garysassano** [comment]
@FidelusAleksander My bad, didn't realize that's how the current system works: I thought the two syntaxes were interchangeable.

**Review by FidelusAleksander — APPROVED** [review]

**Pull request merged**